app/styles/main.scss

```diff
@@ -251,6 +251,14 @@ table {
   }
 }
 
+#communityServiceEmbed{
+  position:absolute;
```

Comment:
Why are absolutely positioning this? It's very rare that doing this is the right solution for a responsive web app. top and left 0 is default for elements anyway. If the embed tag is a block level element, the width is default to 100% already as well.